src/index.js

```diff
@@ -398,6 +401,71 @@ Compute.prototype.createImage = function(name, disk, options, callback) {
   );
 };
 
+/**
+ * Create an instance template.
+ *
```

Comment:
Can you add a `@resource` to https://cloud.google.com/compute/docs/reference/rest/v1/instanceTemplates/insert and https://cloud.google.com/compute/docs/reference/rest/v1/instanceTemplates#resource?